docs/tutorials/15-share-dependencies.md

```diff
@@ -131,7 +131,7 @@ The diagram below shows how this works. Every pilet that uses import maps talks
 
 That way you can safely use multiple versions of your dependencies. If you consolidate them you'll get a performance improvement. Otherwise, you'll get the performance penalty as usual, however, you gained the flexibility to independently choose your version.
 
-The effort to actually use import maps is quite low. The first step is to declare potentially shared dependencies via the pilet's *package.json*:
+The effort to actually use import maps is quite low. The first step is to declare potentially shared dependencies via the pilets *package.json*:
```

Comment:
Not sure why it has been changed here, too. A "pilet" and the "pilet's thing". Its like a "person" and a "person's thing".

Reply:
adatped